sdk/communication/communication-email/CHANGELOG.md

```diff
@@ -1,8 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
+## 1.0.0-beta.2 (2023-03-01)
```

Comment:
Doesn't this need summary of changes?

Reply:
The changes are listed under the "Breaking Changes" section in the full file (since they affect how the customers currently use the SDK).